docs/index.html

```diff
@@ -15,7 +15,7 @@
  </head>
 <body>
 
-<h2>AIS</h2>
+<h2>AISVN</h2>
```

Comment:
The school is now named AISVN for American International School Vietnam to differentiate from the other AIS which is Australian International School.